test/verify/check-client

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/python3 -cimport os, sys; os.execv(os.path.dirname(sys.argv[1]) + "/../common/pywrap", sys.argv)
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2023 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import testlib
+
+
+# Run this on more OSes as we roll out the pybridge
+@testlib.onlyImage("needs pybridge", "debian-testing")
```

Comment:
Could do rawhide and arch now, no?

Reply:
We don't run on rawhide in our CI, and arch was only *really* enabled in the downstream packaging [a few hours ago](https://gitlab.archlinux.org/archlinux/packaging/packages/cockpit/-/commit/296106d68f0c58f3f635936e186f32dbf8569ec1), so we still need to wait for the next image refresh. But yes, I want to run this on more OSes obviously. Looking forward to c9s, too!